.travis.yml

```diff
@@ -1,3 +1,5 @@
+sudo: required
```

Comment:
As you wrote here: https://github.com/damianszczepanik/cucumber-reporting/pull/127#issuecomment-128846165 coverage uploader doesn't work properly on container-based infrastructure. Without specifing that sudo is required travis [routes builds of linux projects that were added after 01.01.2015 to container-based infrastructure](http://docs.travis-ci.com/user/workers/container-based-infrastructure/), so it won't work for new contibutors' forks of the project. You're right that it isn't related to my code changes, so if you prefer i will remove it.